deps/oob-1.0.2/jsonutil.c

```diff
@@ -404,7 +410,35 @@ void setoobConnectioninfo(const char* host,
     copyString("https", connectionInfo.protocol, 8);
   else
     copyString("http", connectionInfo.protocol, 8);
+}
 
+void setoobDsninfo(const char *key, const char *val) {
+    char* caps = calloc(strlen(key), sizeof(char));
+    upper(caps, key);
+
+    if (dsn == NULL) {
+        dsn = cJSON_CreateObject();
+    }
+    cJSON *value = cJSON_CreateString(val);
+    if (strcmp(caps, "PWD")) {
```

Comment:
You shouldn't be doing this here. ODBC maintains a list of "sensitive keys" and you should make sure you don't pass in the real value for this in the first place.